apps/music/elements/music-search-box.js

```diff
@@ -200,14 +159,4 @@ function debounce(fn, ms) {
   };
 }
 
-try {
```

Comment:
I can't see what value this adds. You're catching the error and then re-throwing with no additional messaging or logic. In the case of `NotSupportedError`, execution will still be interrupted and we'll see the same Error in the console with or without the `try catch`